test/common/alerts_exist.go

```diff
@@ -239,6 +239,157 @@ var expectedRules = []alertsTestRule{
 		File:  "redhat-rhmi-amq-online-enmasse-console-rules.yaml",
 		Rules: []string{},
 	},
+	{
+		File: NamespacePrefix + "apicurito-ksm-endpoint-alerts.yaml",
+		Rules: []string{
+			"RHMIApicuritoServiceEndpointDown",
```

Comment:
@Nanyte25 @alanmoran fyi, ~50 new alerts to cover all Services having at least 1 Endpoint Address available.
They all use a similar query, and will all have the same SOP.
The reason for having individual alerts rather than 1 generic one was to allow for a couple of edge cases where a generic alert would be tricky to allow for (& maintain), and to allow tweaking individually per product & per service in future.